cloud-service-providers/azure/aks/README.md

```diff
@@ -0,0 +1,17 @@
+# nim on aks
+
+AKS need to be properly created in order to be able deploy NIM on it.  There are two things need to be taken care of.  Make sure you have right GPU for NIM and right driver version as well.  Unfortunately, default GPU driver in AKS usually always a little bit too old for latest NVidia software in general.  Microsoft has no official solution for it yet.  There is a preview version of CLI to solve this issue.  We need to create AKS using preview version of CLI.  Prerequisite section talks about how to set up your local environment to enable you to create AKS from preview version of CLI.
```

Comment:
Can we spell out what AKS means at first mention? Is that usually how Azure docs work?

Reply:
done